hydroflow_macro/Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "hydroflow_macro"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+default = []
+diagnostics = [ "hydroflow_lang/diagnostics" ]
```

Comment:
So because this is a proc macro, diagnostics should just always be on rather than behind a propagated feature.